integration/runtime_test.go

```diff
@@ -156,6 +156,8 @@ func spawnGlobalDaemon() {
 	globalEngine = eng
```

Comment:
I think no. I'll remove it. Just didn't wanna to touch awful integartion tests :)